Setup/InstallSchema.php

```diff
@@ -55,7 +55,7 @@ public function install(// @codingStandardsIgnoreLine
     ) {
         $setup->startSetup();
         $this->createCustomerTable($setup);
-        $this->createProductQueueTable($setup);
+        $this->createProductCacheTable($setup);
```

Comment:
Does the upgrade work? Have we tested it?

Reply:
Yes, it works